### ISSUE TITLE
Error: config key invalid: DontCheckOSXFUSE %!s(bool=true)

### ISSUE THREAD

**Issue by maedoc** [ipfs/go-ipfs#3612]
#### Version information:
```
go-ipfs version: 0.4.4-
Repo version: 4
System version: amd64/darwin
Golang version: go1.7
```

#### Description:

Having difficulty getting `mount` to work, 

```bash
$ ipfs mount
Error: unable to check fuse version.
...
$ go get github.com/jbenet/go-fuse-version/fuse-version
$ ipfs mount
Error: unable to check fuse version.
...
$ fuse-version
OSXFUSE.FuseVersion: 29
OSXFUSE.AgentVersion: 3.5.4
OSXFUSE.AgentName: OSXFUSE
$ ipfs mount
Error: unable to check fuse version.
...
$ ipfs config DontCheckOSXFUSE true
$ ipfs mount
Error: config key invalid: DontCheckOSXFUSE %!s(bool=true)
```
yet, in `~/.ipfs/config` there are other `true` values so I'm not sure what's going on.

**Comment by whyrusleeping** [comment]
Try doing:
```
ipfs config --json DontCheckOSXFUSE true
```

That *should* fix the issue... This behaviour has been improved in 0.4.5 and what you tried should work on 0.4.5-dev

**Comment by maedoc** [comment]
I still get the above `Error: config key invalid`. I'll try again later on 0.4.5.

**Comment by whyrusleeping** [comment]
I confirmed that this is fixed in (at least) 0.4.10 on OSX
